website/static/figure.js

```diff
@@ -137,6 +137,27 @@ class ZFigure {
 
 		return this;
 	}
+	add_warning_overlay() {
+		this.svg.append("rect")
+			.attr("x", -30)
+			.attr("y", -30)
+			.attr("width", this.WIDTH + 100)
+			.attr("height", this.HEIGHT + 100)
+			.attr("fill", "white")
+			.attr("pointer-events", "all")
+			.attr("id", "overlay")
+		this.svg.append("text")
+			.attr("x", 200)
+			.attr("y", 100)
+			.attr("font-size", "20px")
+			.attr("id", "warning_text")
+			.text("Enrichment track unavailable for this hydrophobicity scale");
```

Comment:
More precisely: "Enrichment predictions are only available for the Kyte-Doolittle scale at this time."